docs/configuring.md

```diff
@@ -490,6 +490,33 @@ Overwrite files that already exist, without asking.
 Also don't ask questions to the user; just use default values
 [obtained from other sources](#configuration-sources).
 
+!!! info
+
+    Not supported in `copier.yml`.
+
+### `defaults`
+
+-   Format: `bool`
+-   CLI flags: `--defaults`
+-   Default value: `False`
+
+Use default answers to questions, which might be null if not specified.
+
+!!! info
+
+    Not supported in `copier.yml`.
+
+### `overwrite`
+
+-   Format: `bool`
+-   CLI flags: `--overwrite`
+-   Default value: `False`
+
+Overwrite files that already exist, without asking.
+
+Also don't ask questions to the user; just use default values
```

Comment:
I think this statement is wrong.

Reply:
Seems it was copied from the `force` description.

Reply:
yes, copy/paste error.